client/blocks/inline-help/inline-help-rich-result.jsx

```diff
@@ -50,15 +41,12 @@ class InlineHelpRichResult extends Component {
 		const isLocaleEnglish = 'en' === getLocaleSlug();
 		const { type, tour, link, searchQuery, postId } = this.props;
 
-		const tracksData = omitBy(
-			{
-				search_query: searchQuery,
-				tour,
-				result_url: link,
-				location: 'inline-help-popover',
-			},
-			( data ) => typeof data === 'undefined'
-		);
+		const tracksData = {
+			search_query: searchQuery,
+			location: 'inline-help-popover',
+			...( link && { result_url: link } ),
+			...( tour && { tour } ),
+		};
```

Comment:
The `omitBy` can be avoided completely because `recordTracksEvent` already does that internally: https://github.com/Automattic/wp-calypso/blob/trunk/packages/calypso-analytics/src/tracks.ts#L232

It's safe to pass `undefined` values.

Reply:
Addressed in https://github.com/Automattic/wp-calypso/pull/56063/commits/1c8ac82a5b9bdff81f741390deeeae467ba3d8b0